OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+* Creating WebCenter Content domain
+* Extending WebCenter Content domain for associated products (e.g. IPM, ADFUI etc.) - based on **component** env varibale. 
+* Configuring Node Manager
+* Starting Node Manager
+* Starting Admin Server
+
+#### B. Stopping Admin Container
+```
+docker container stop WCCAdminContainer
+```
+
+#### C. Starting Admin Container
+```
+docker container start -i WCCAdminContainer
+```
+
+### 5.1.3. Managed Server Container (WCContentContainer)
+
+#### A. Creating and Running WCContent Container
+Run the following command to create the WCContent Managed Server container:
+
+```
+docker run -it --name WCContentContainer --network=WCContentNET -p <UCM_HOST_PORT>:<UCM_PORT> -p <IBR_HOST_PORT>:<IBR_PORT> -p <UCM_INTRADOC_PORT>:<UCM_INTRADOC_PORT> -p <IBR_INTRADOC_PORT>:<IBR_INTRADOC_PORT> --volumes-from WCCAdminContainer --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0  /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterContent.sh; /bin/bash"
+
+# A sample command will look like this -
+
+docker run -it --name WCContentContainer --network=WCContentNET -p 16200:16200 -p 16250:16250 -p 4444:4444 -p 5555:5555 --volumes-from WCCAdminContainer --env-file /scratch/wccdocker/webcenter.env.list oracle/wccontent:12.2.1.4.0  /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterContent.sh; /bin/bash"
+```
+The docker run command creates the container as well as starts the WebCenter Content managed servers. 
+
+Note: 1. If manged servers need to be accessed through host ports different from container ports, then intended host port values needs to be supplied as part of -p option of the 'docker run'
+command mentoned above (for ex. -p 16201:16200 and -p 16251:16250). The same port value needs to be updated in the webcenter.env.list as UCM_HOST_PORT and IBR_HOST_PORT.
+If managed servers are going to be accessed via same host port number as the container port, then UCM_PORT and UCM_HOST_PORT values (and IBR_PORT and IBR_HOST_PORT) should be same in the webcenter.env.list.  
+      2. Intradoc ports are for internal server communications and not meant for browser access. While, intradoc ports on container are configurable (like other parametres like admin credentials, admin port, domain-name, manged server container ports) 
+through webcenter.env.list, publishing these to different host ports is not supported. This essentially means one can provide '-p 7777:7777' instead of '-p 4444:4444', but '-p 6666:7777' is not supported.
+
+#### B. Stopping WCContent Container
+```
+docker container stop WCContentContainer
+```
+
+#### C. Starting WCContent Container
+```
+docker container start -i WCContentContainer
+```
+
+#### D. Getting Shell in WCContent Container
+```
+docker exec -it WCContentContainer /bin/bash
+```
+Please wait till all the above all 'docker run'commands are successfully completed, before you can start Admin and Managed Servers with weblogic admin credentials.
```

Comment:
Missing a space between 'docker run' and "commands". Also, `docker run` should be wrapped in back ticks not apostrophes. WebLogic should also have a capital W and L as it is a product name.